react/Infos/index.jsx

```diff
@@ -27,7 +62,13 @@ Infos.defaultProps = {
 }
 
 Infos.propTypes = {
+  /** An button that will be render in the infos component */
+  actionButton: PropTypes.element,
+  /** An icon to display at the left of the text */
   icon: iconPropType,
-  text: PropTypes.string
+  /** An important information will be displayed with some red colors */
```

Comment:
```suggestion
  /** An important information will be displayed with red colors */
```

I think we should removed "some" as it is a vague word without much information. IMHO, technical writing should be direct and to the point.